website/models.py

```diff
@@ -312,6 +313,22 @@ def remove_user(self):
     def get_absolute_url(self):
         return "/issue/" + str(self.id)
 
+    def get_cve_score(self):
+        if self.cve_id:
+            try:
+                url = "https://services.nvd.nist.gov/rest/json/cves/2.0?cveId=%s" % (self.cve_id)
+                response = requests.get(url).json()
+                results = response["resultsPerPage"]
+                if results != 0:
+                    metrics = response["vulnerabilities"][0]["cve"]["metrics"]
+                    if metrics:
+                        cvss_metric_v = next(iter(metrics))
+                        score = metrics[cvss_metric_v][0]["cvssData"]["baseScore"]
+                        self.score = score
```

Comment:
It should either return the `score` or set it, not both. Based on the function name I'd go with return option.